docs/guides/references/changelog.mdx

```diff
@@ -2,6 +2,41 @@
 title: Changelog
 ---
 
+## 12.14.0
+
+_Released 06/07/2023_
+
+**Features:**
+
+- A new testing type switcher has been added to the Spec Explorer to make it
+  easier to move between E2E and Component Testing. An informational overview of
+  each type is displayed if it isn't configured in your project to help
+  newcomers to each testing type. Addresses
+  [#26448](https://github.com/cypress-io/cypress/issues/26448),
+  [#26836](https://github.com/cypress-io/cypress/issues/26836) and
+  [#26837](https://github.com/cypress-io/cypress/issues/26837).
+
+**Bugfixes:**
+
+- Fixed an issue to now correctly detect Angular 16 dependencies during
```

Comment:
add npm link to angular npm package

Reply:
@emilyrohrbough This fix is for 5 different dependencies. Should the Changelog list them all out, or can the user just click through to the issue to see which ones have been corrected?

Reply:
I personally expect the change log to surface this information. users should never need to click the issues/pr links to understand what changes. but this is probably fine as is. I would expect a link to angular

Reply:
Nevermind, I reread the guide about not making users click through issues to understand the change log.  I will add the appropriate links.